spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -68,5 +68,13 @@
 
       expect(rendered).to have_link('Last Updated', href: href)
     end
+
+    it 'has a link to the content item details page' do
+      render
+      expect(rendered).to have_link('View detail', href: organisation_content_item_path(
```

Comment:
Very minor: Could you please leave an empty line between the `render` and the expectation?